README.md

```diff
@@ -49,13 +51,13 @@ my_logger = logging.getLogger('my_project')
 log_handler = RotatingFileHandler(r'c:\log.log'), backupCount=10)
 my_logger.addHandler(log_handler)
 
-supervisor = Supervisor(logger=my_logger, log_path=r'c:\log.log')
+supervisor = Supervisor(handle_errors=True, logger=my_logger, log_path=r'c:\log.log')
```

Comment:
**question**: this is the default correct and does not need to be supplied? Only if we want to stop handling errors do we need to pass false.

Reply:
Correct. This preserves the original behavior without any changes to client code.